frontend/src/pages/index.vue

```diff
@@ -1,3 +1,9 @@
+<template>
+  <div>
+    <h1>Welcome to LeadMiner</h1>
```

Comment:
Remove welcome message, keep an empty div since this page is meant only for redirection.